tests/unit_tests/include/test_helper.hpp

```diff
@@ -67,11 +67,18 @@
 #define TEST_RUN_INTELGPU_SELECT(q, func, ...)
 #endif
 
+/// TODO: Combine into a single CUDA test?
```

Comment:
Should we remove TODO comments before merge?

Reply:
For sure. I will fix up and ping when ready.